astrophot/image/image_header.py

```diff
@@ -463,42 +301,58 @@ def get_state(self):
         return state
 
     def _save_image_list(self):
+        """
+        Constructs a fits header object which has the necessary information to recreate the Image_Header object.
+        """
         img_header = fits.Header()
         img_header["IMAGE"] = "PRIMARY"
-        img_header["PXLSCALE"] = str(self.pixelscale.detach().cpu().tolist())
         img_header["WINDOW"] = str(self.window.get_state())
         if not self.zeropoint is None:
             img_header["ZEROPNT"] = str(self.zeropoint.detach().cpu().item())
         if not self.note is None:
             img_header["NOTE"] = str(self.note)
         return img_header
 
+    def set_fits_state(self, state):
+        """
+        Updates the state of the Image_Header using information saved in a fits header.
+        """
+        self.zeropoint = eval(state.get("ZEROPNT", "None"))
+        self.note = state.get("NOTE",None)
```

Comment:
```suggestion
        self.note = state.get("NOTE", None)
```

Sorry, I don't know if you're using a formatting style here.  I'm just so used to reading code where there are always spaces after commas.

Your call.  Honestly, my real answer is that I've switched to just using `black` formatting and I never explicitly think about having to writing any of this formatting.

Reply:
Right, I still need to run black on everything. It will make the decision for me